st_louis/bills.py

```diff
@@ -0,0 +1,132 @@
+from pupa.scrape import Scraper
+from pupa.scrape import Bill
+from pupa.scrape import BaseBillScraper
+from .utils import Utils
+import time
+
+
+class StLouisBillScraper(Scraper):
+
+	def scrape(self):
+		for session in self.jurisdiction.legislative_sessions:
+			session_id = session["identifier"]
+			session_url = self.bill_session_url(session_id)
+			page = Utils.lxmlize(session_url)
```

Comment:
If you used defined lxmlize like this https://github.com/opencivicdata/scrapers-us-municipal/blob/master/chicago/legistar.py#L18, where the requests are using the `Scrapers` class method than you get a lot of things for free, including caching.